docker/Makefile.image_names

```diff
@@ -14,26 +15,39 @@ FV3CORE_DOCKERFILE = Dockerfile
 # fv3core or fv3core-wrapper
 FV3CORE_TARGET ?= fv3core
 
-# base images w/ or w/o CUDA
+BASE_IMAGE = ubuntu:18.04
+# If trying to test the gtcuda backend, override the CUDA option
+# and use the cuda based image
+ifneq (,$(findstring gtcuda,$(TEST_ARGS)))
+    CUDA_FLAGS = --gpus all
+    CUDA = y
+else
+    CUDA_FLAGS =
+endif
+
+
+# base environment images w/ or w/o CUDA
 ifeq ($(CUDA),n)
-	BASE_IMAGE = ubuntu:19.10
-	TAG_NAME = $(VERSION)-gnu9-mpich314-nocuda-develop
+	BASE_IMAGE_ENV = ubuntu:18.04
+	TAG_NAME = $(VERSION)-gnu7-mpich314-nocuda-develop
```

Comment:
Should this be `gnu8`?

Reply:
yes good point, though gfortran-8 inside of the docker image seems to be breaking the parallel tests through sarus on daint... I am trying using only gfortran-7 and gcc-8, g++-8... if that still doesn't work, is it ok if I revert back to gnu-7 and leave updated to gnu 8 for the step that would replace our sarus tests with testing using your venv?

Reply:
Actually -- should I just turn off daint tests? since we've decided to not support containers there, there's no reason to keep running these there if we are going to very shortly start running tests 'on metal' using your venv